cluster/images/hyperkube/static-pods/etcd.json

```diff
@@ -10,7 +10,7 @@
     "containers": [
       {
         "name": "etcd",
-        "image": "gcr.io/google_containers/etcd-ARCH:2.3.7",
+        "image": "gcr.io/google_containers/etcd-ARCH:2.2.1",
```

Comment:
This file is only for the docker/docker-multinode setup. 
It's 100% safe to just bump this to 2.2.5 as it was, as upgrades/downgrades aren't a thing for that deployment.
We've used 2.2.5 quite a long time now, and should continue to do that for docker-multinode